### DISCUSSION TITLE
Is it possible to disable the help at the bottom, and to customize the tabline?

### DISCUSSION THREAD

**Discussion by abhikjain360** [zellij-org/zellij#425] [Q&A] [answered]
When using zellij with a neovim, the help line at the bottom becomes particularly distracting with all it's bright colors, and because I use plugins with neovim for tabline, there are 2 tablines on the top and 2 status/help lines at the bottom which makes is hard to use zellij with neovim.

Is there are way to disable the help bar at the bottom, and to customize the tabline at the top so that it is minimal and less distracting?

Thanks for you help!

**Comment by a-kenji** [comment, marked as answer]
Hello! The status-bar and tab-bar are both plugins, you can customize your default [layout](https://zellij.dev/documentation/layouts.html) to fit your needs, eg. remove the status-bar plugin from it.

There is no support for customizing plugins yet, it is definitely something that is on the roadmap. But it still will take some time.
